server/src/main/scala/za/co/absa/atum/web/model/Partitioning.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.web.model
+
+import za.co.absa.atum.model.dto.PartitionDTO
+
+case class Partitioning (
```

Comment:
**Minor:**
Would be considering naming the class `DBPartioning` or `PartiioningForDB` to avoid confusion with Agent class

Reply:
Sure, good idea actually 👍